smoke-test-pipeline.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 schedules:
 - cron: "0 3 * * *"
   branches:
-    include: [ master ] 
+    include: [ master, nate.infra.integration.scaffold ] 
```

Comment:
@NathanielRose why run off of a separate branch?

Reply:
Being used for testing, will be removed on final merge.

Reply:
@yradsmikham @NathanielRose please remove